tests/Behat/Context/FixtureContext.php

```diff
@@ -74,4 +74,43 @@ public function contentBlocksAreNotInLineEditable()
 
         $this->getMainContext()->visit('/?flush');
     }
+
+    /**
+     * @Given the page template includes elemental blocks
+     */
+    public function pageTemplateIncludesElementalBlocks()
+    {
+        $template = <<<'TEMPL'
+<h1>$Title</h1>
+<div class="content">
+    $Content
+    $ElementalArea
+</div>
+TEMPL;
+
+        $templFile = 'Layout/Page.ss';
+        $templPath = $this->getDestinationTemplatePath($templFile, true);
+        file_put_contents($templPath, $template);
+
+    $themeConfig = <<<'CONF'
+---
+Name: mytheme-override
+Before: mytheme
+---
+SilverStripe\View\SSViewer:
```

Comment:
Is something like this done elsewhere in behat?  This seems wrong, can't we just update config like normal?

Reply:
By "something like this" do you mean adding config files? If so, yes - literally in the same file this is done for other config.